docs/config.md

```diff
@@ -101,6 +101,10 @@ Callables or other Python objects have to be passed in `setup.py` (via the `use_
         Setuptools will still normalize it to create the final distribution,
         so as to stay compliant with the python packaging standards.
 
+`disable_i18n`
```

Comment:
having it a config options is fundamentally wrong as everyone affected would have to enable it individually